drip/models.py

```diff
@@ -79,12 +93,13 @@ class SentDrip(models.Model):
         on_delete=models.CASCADE,
     )
     user = models.ForeignKey(
-        getattr(settings, 'AUTH_USER_MODEL', 'auth.User'),
+        getattr(settings, 'DRIP_CAMPAIGN_USER_MODEL', 'auth.User'),
```

Comment:
shouldn't we default to the whatever user the application is using instead of `auth.User`?

Reply:
@kaozdl I have a case where I need to do the drip campaign for a model that is not the User model
So this way, I can tell drip campaign to deal with the model I want without missing the User model

Reply:
I see why that's useful, in any case shouldn't you default then to 'AUTH_USER_MODEL'?
if you default to 'auth.User', any custom configuration in any other project will be lost.

Reply:
@kaozdl Ohh I see your point, yeah you're right I will do that. Thanks